### DISCUSSION TITLE
Wheel for Goes2Go

### DISCUSSION THREAD

**Discussion by bryanguarente** [blaylockbk/goes2go#68] [Ideas]
Brian,
I saw your post on LinkedIn the other day where you were building a pyscript dashboard for your SynopticAPI work.  I have been working on dashboards for Goes2Go and Herbie for use at COMET (and maybe larger).  I have been doing that in Jupyter Notebooks, but you inspired me with the pyscript work and now I am trying to push those dashboards online to see what happens.  

In that vein though, it looks like you don't have pypi wheel files for goes2go (honestly, I didn't check herbie).  Is that something that is easy for you to update?  I would love to mess with this in a javascript environment (probably my favorite language).

No rush by any means here, just interested to play along with new developments.

Thanks,
Bryan

**Comment by blaylockbk** [comment]
I'll have to look into the wheels, but even after providing those there would probably have to be some re-writing of goes2go and herbie because [pyscript can't use the `requests` package](https://docs.pyscript.net/latest/guides/http-requests.html).

Pyscript was fun and fairly easy to figure out. Someday I would like to rebuild my [custom HRRR plotting tool](https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/hrrr_custom.html) using pyscript.

**Comment by blaylockbk** [comment]
I'm not too knowledgeable on best-practices for packaging, but isn't this the wheel file?
https://pypi.org/project/goes2go/#files

![image](https://github.com/blaylockbk/goes2go/assets/6249613/8ffdd249-be60-4d7c-8f77-f19ffc52a05e)

**Reply by bryanguarente** [reply]
You're right, it is there.  I imagine the problem is the requests like you have stated but the error message provided is incorrect for this use case.  Thanks for your concern and trying to find a solution.  

Feel free to close this out.
Bryan
